app/src/main/java/com/eveningoutpost/dexdrip/utils/LibreTrendUtil.java

```diff
@@ -82,71 +115,71 @@ boolean IsTimeValid(long timeId) {
         return timeId >= 0 && timeId < MAX_POINTS;
     }
     
-    List<LibreTrendPoint> getData(long startTimestamp, long endTimestamp) {
-        Log.i(TAG, "Size of array is " + m_points.size() + " this = " + this);
+    public List<LibreTrendPoint> getData(long startTimestamp, long endTimestamp) {
+        Log.i(TAG, "Size of array is " + m_points.size() + " this = " + this + "m_libreTrendLatest.timestamp " + m_libreTrendLatest.timestamp);
```

Comment:
Maybe you should use a separator (space or colon) before "m_libreTrendLatest.timestamp".

Reply:
Done.